package.json

```diff
@@ -129,11 +129,12 @@
     "@noble/hashes": "1.3.2",
     "@noble/secp256k1": "2.0.0",
     "@octokit/types": "12.0.0",
-    "@radix-ui/colors": "2.1.0",
+    "@radix-ui/colors": "3.0.0",
     "@radix-ui/react-accessible-icon": "1.0.3",
+    "@radix-ui/react-dialog": "1.0.5",
     "@radix-ui/react-switch": "1.0.3",
     "@radix-ui/react-tabs": "1.0.4",
```

Comment:
Did we end up not using `react-switch` and `react-tabs` bc searching the codebase I'm not finding use of them?

Reply:
Good eyes 👁️ . I checked and we use radix tabs via `radix-ui/themes` so we can probably ditch both of these imports